src/cli.js

```diff
@@ -35,7 +35,7 @@ Options:
   --v8-cache               Emit a build using the v8 compile cache
   --license [file]         Adds a file containing licensing information to the output
   --stats-out [file]       Emit webpack stats as json to the specified output file
-  --webpack-target         What build target to use for webpack (default: node)
+  --target                  What build target to use for webpack (default: es6)
```

Comment:
Where do you see default is `es6`? Is that documented by webpack somewhere?

Reply:
It's the default for webpack `v5` with `node` https://github.com/webpack/webpack/blob/7415a618460795874fdf95c8f03363ea96709d52/lib/config/target.js#L163-L171